src/runtime/CollectionWrappers/ListWrapper.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Collections.Generic;
+
+namespace Python.Runtime.CollectionWrappers
+{
+    internal class ListWrapper<T> : SequenceWrapper<T>, IList<T>
+    {
+        public ListWrapper(PyObject pyObj) : base(pyObj)
+        {
+
+        }
+
+        public T this[int index]
+        {
+            get
+            {
+                IntPtr item = Runtime.PySequence_GetItem(pyObject.Handle, index);
```

Comment:
I wonder if there would be any difference between using `PySequence_GetItem` vs `PyList_GetItem` vs `PyObject_GetItem` here.

Reply:
@lostmsu good point.  I should use PyList_GetItem in ListWrapper and PySequence_GetItem in SequenceWrapper.